server/routes/main.js

```
@@ -220,6 +230,60 @@ router.post('/search', async (req, res) => {
+    }
+
+    if(!commenterName ||!commentBody) {
+        console.error(401, 'Invalid comment data');
+        return res.redirect(`/post/${postId}`);
+    }
+    if(commentBody.length > 500 || commenterName.length > 50 || commenterName.length < 3 || commentBody.length < 1) {
+        console.error(401, 'Invalid comment data', 'Size mismatch');
+        return res.status(401).json({"status": "401", "message": "Invalid comment data" });
+    }
+
+    const post = await post.findById(postId);
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)